test/fixtures/get-test-server.ts

```diff
@@ -25,9 +25,10 @@ interface ApiRequest extends NextApiRequest {
 }
 
 export const getTestServer = async (
-  t: ExecutionContext
+  t: ExecutionContext,
+  { seed = true }: { seed?: boolean } = {}
```

Comment:
Why was this added? Is there a use case for testing with an unseeded DB? I would prefer a separate function `getUnseededTestServer` in that case (then you can avoid the type issues 😄).

Reply:
there's supported to be a rule around only having one getTestServer for london-style testing so that the test always look the same, this is basically a seam connect convention here to provide the seed parameter. Yea the type issue is annoying but I feel like it's just a temporary intentional bug since typing this function is possible, I'm just trying to not get fancy atm

Reply:
I can fix the type issue if you think it's worth blocking

Reply:
Many of the changes we are adding here might be back-ported to the template, that is why I'm flagging things like this. I guess I still am unsure what is going to use this param. 

If you have a better solution for the types, that would be nice. Not understanding the use case is what is making it hard for me to make a call. If people will write test with this, I think they should be confident in the types and not get null errors.

Reply:
I think sometimes it is nice to have a blank fake, especially when doing security-access testing

Reply:
imo too complex (it might make sense if it was outside a test suite to get complex, but idk) 5593413